### DISCUSSION TITLE
Why pdm init default tmeplate is not minimal?

### DISCUSSION THREAD

**Discussion by ryomahan** [pdm-project/pdm#3227] [Q&A] [answered]
Hi, @frostming  

Either `npm init` or `pnpm init` only output a package.json file, but pdm init default template includes something more. It's counter-intuitive, so:
1.  why was it set up that way in the begining?
2. can we have a `pdm init` argument to use minimal template? because we can't find minimal by `pdm init -h` and it also can't simply be found on the doc website. I'd love to do this.

**Comment by ryomahan** [comment]
or maybe we can make user select in the init step, another way is move template feature to a new command like `npm create`, `pnpm create`.

**Comment by frostming** [comment]
> can we have a pdm init argument to use minimal template? 

of course， `pdm init minimal`

**Reply by ryomahan** [reply]
yeah, I find this command use at least five minutes, as I say 
> because we can't find minimal by pdm init -h and it also can't simply be found on the doc website.

the easiest way is  print this in `pdm init -h`

**Reply by frostming** [reply, marked as answer]
Oh sure. 0b7e5fa9e9b3b2cc0653a57bde55c887b06343b6
